pkg/proxy/ipvs/proxier.go

```diff
@@ -1257,7 +1257,12 @@ func (proxier *Proxier) syncProxyRules() {
 			if err := proxier.syncService(svcNameString, serv, true); err == nil {
 				activeIPVSServices[serv.String()] = true
 				activeBindAddrs[serv.Address.String()] = true
-				if err := proxier.syncEndpoint(svcName, false, serv); err != nil {
+
+				onlyNodeLocalEndpoints := false
+				if utilfeature.DefaultFeatureGate.Enabled(features.ExternalPolicyForExternalIP) {
```

Comment:
Same question for IPVS, on line 1240 we're adding the externalIP to the ipset `kubeExternalIPSet` which we SNAT. Should there be a new ipset like `kubeExternalIPNodeLocalIPSet` that skips SNAT?

Reply:
Fixed. Added `kubeExternalIPNodeLocalIPSet ` that skips SNAT